docs/dex-specification.md

```diff
@@ -656,6 +656,8 @@ Long-term staking allows users to stake the Dex token and earn other tokens as r
 
 The contract deploys a new staking pool contract using [`deployPool`](#deploypool). The created contract is owned by the [`multisig` contract](#multisignature-wallet).
 
+The `StakingFactory` contract is responsible for deploying new [`StakingInitializable`](#stakinginitializable) contracts. The factory fully initializes the new contract after its deployment in `deployPool`, but the reward tokens have to be sent to the pool manually after its deployment.
```

Comment:
it's correct, but `after its deployment` is duplicated at the end of both sentences, sounds heavy, maybe rephrase it to smth like:
The factory fully initializes the new pool contract after its deployment in `deployPool`, but the reward tokens **have to be sent** to the initialized pool **manually**.
_Note: Since in the first statement we already mean that the pool is initialized only after its deployment, so it means that the tokens are to be sent there only after deployment by default. So, I guess, we can rephrase it _